dotCMS/src/main/java/com/dotmarketing/common/reindex/ReindexThread.java

```diff
@@ -323,4 +328,7 @@ protected void sendNotification(final String key, final Object[] msgParams, fina
         systemUser.getLocale());
   }
 
+    public static void setCurrentIndexReadOnly(boolean currentIndexReadOnly) {
```

Comment:
set to final

Reply:
done https://github.com/dotCMS/core/pull/18604/commits/fc979e0e1d04475951dbb213b379b17022474a8e#diff-6aad54df22d1ffe94dc4adf9a1d98637R331